### PR TITLE
Add release notes for 1.14.1

### PR DESCRIPTION
**Pull request by jrhee17**

### CODE REVIEW

File: site/src/pages/release-notes/1.14.1.mdx [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+date: 2022-2-16
+---
+
+## 🌟 New features
+
+- N/A
+
+## 📈 Improvements
+
+- Users have more transparency on why they encounter an `EmptyEndpointGroupException` #4084
```
Comment (trustin) [inline site/src/pages/release-notes/1.14.1.mdx:11]: Missing period in a sentence?

File: site/src/pages/release-notes/1.14.1.mdx [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+## 🛠️ Bug fixes
+
+- `DnsEndpointGroup` now correctly respects a trailer dot of a hostname when resolving DNS records. #4088
```
Comment (trustin) [inline site/src/pages/release-notes/1.14.1.mdx:15]: a trailing dot in a hostname

File: site/src/pages/release-notes/1.14.1.mdx [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+- You can disable the ignored multiple routes warning log by specifying `-Dcom.linecorp.armeria.reportIgnoredMultipleRoutes=false`. #4086
+- `GrpcClientBuilder` now correctly normalizes a path prefix. #4085
+- `GrpcClientBuilder.path()` has been deprecated in favor of `GrpcClientBuilder.pathPrefix()`. #4085
```
Comment (trustin) [inline site/src/pages/release-notes/1.14.1.mdx:18]: Is this a bug or deprecation?

File: site/src/pages/release-notes/1.14.1.mdx [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+- N/A
+
+## 🙇 Thank you
+
+<ThankYou usernames={[
+  'heowc',
+  'ikhoon',
+  'jrhee17',
+  'minwoox',
+  'tobias-'
```
Comment (trustin) [inline site/src/pages/release-notes/1.14.1.mdx:45]: I reviewed 😆

File: site/src/pages/release-notes/1.14.1.mdx [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+
+- Users have more transparency on why they encounter an `EmptyEndpointGroupException` #4084
+
+## 🛠️ Bug fixes
+
+- `DnsEndpointGroup` now correctly respects a trailer dot of a hostname when resolving DNS records. #4088
+- You can disable the ignored multiple routes warning log by specifying `-Dcom.linecorp.armeria.reportIgnoredMultipleRoutes=false`. #4086
+- `GrpcClientBuilder` now correctly normalizes a path prefix. #4085
+- `GrpcClientBuilder.path()` has been deprecated in favor of `GrpcClientBuilder.pathPrefix()`. #4085
+- Server will stop responding 500 when sending null GraphQL variable #4068 #4069
```
Comment (trustin) [inline site/src/pages/release-notes/1.14.1.mdx:19]: null -> `null`

File: site/src/pages/release-notes/1.14.1.mdx [hunk 6]
```diff
@@ -0,0 +1,51 @@
...
+- N/A
+
+## 📈 Improvements
+
+- Context of the `EndpointGroup` is added to the `EmptyEndpointGroupException` message. #4084
+
+## 🛠️ Bug fixes
+
+- `DnsEndpointGroup` now correctly respects the trailing dot in a hostname when resolving DNS records. #4088
+- You can now allow double dots (`..`) in query string by specifying the
```
Comment (trustin) [inline site/src/pages/release-notes/1.14.1.mdx:16]: in a request path query string

File: site/src/pages/release-notes/1.14.1.mdx [hunk 7]
```diff
@@ -0,0 +1,51 @@
...
+  `-Dcom.linecorp.armeria.allowDoubleDotsInQueryString=true` JVM option. #4091 #4094
+- `GrpcClientBuilder` now correctly normalizes path prefixes. #4085
+- `GraphQLService` won't reject GraphQL requests with `null` variables. #4068 #4069
```
Comment (trustin) [inline site/src/pages/release-notes/1.14.1.mdx:19]: `<type://GraphqlService>`

File: site/src/pages/release-notes/1.14.1.mdx [hunk 8]
```diff
@@ -0,0 +1,51 @@
...
+
+- N/A
+
+## ⛓ Dependencies
+
+- N/A
+
+## 🗑 Maybe ignore
+
+- N/A
```
Comment (trustin) [inline site/src/pages/release-notes/1.14.1.mdx:39]: Could remove these sections

**Comment by jrhee17** [comment]
Screen Shot 2022-02-18 at 3 32 56 PM

File: site/src/pages/release-notes/1.14.1.mdx [hunk 9]
```diff
@@ -0,0 +1,31 @@
...
+date: 2022-2-18
+---
+
+## 📈 Improvements
+
+- Context of the <type://EndpointGroup> is added to the <type://EmptyEndpointGroupException> message. #4084
+
+## 🛠️ Bug fixes
+
+- `DnsEndpointGroup` now correctly respects the trailing dot in a hostname when resolving DNS records. #4088
```
Comment (minwoox) [inline site/src/pages/release-notes/1.14.1.mdx:11]: `<type://DnsEndpointGroup>`? 😄

**Review by minwoox — APPROVED** [review]
Thanks, @jrhee17!

File: site/src/pages/release-notes/1.14.1.mdx (hunk 9 above)
Comment (minwoox) [reply site/src/pages/release-notes/1.14.1.mdx]: I thought it was public. 😅

File: site/src/pages/release-notes/1.14.1.mdx (hunk 3 above)
Comment (jrhee17) [reply site/src/pages/release-notes/1.14.1.mdx]: I think #4085 did both (bug fix + deprecation), so I added the PR in each section.
Previously, the message was duplicated though, so I fixed it 👍

**Review by trustin — APPROVED** [review]
Thanks, @jrhee17 🙇

**Comment by minwoox** [comment]
Thanks a lot, @jrhee17! 😄

**Pull request merged**